src/components/Icons.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+interface IconProps {
+  className?: string
+}
+
+export const CorrectAnswerIcon: React.FC<IconProps> = ({ className }) => (
```

Comment:
Super minor: This declaration is somewhat inconsistent in this code base.

Reply:
Updated per this [commit](https://github.com/openstax/k12-apps-raise/pull/89/commits/040f8931d9c2d6451b0d23a1920bccde6a4843cd).